app/javascript/components/StudioAssessmentQuestionView/index.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
```

Comment:
is there a way you can use StudioAssessmentQuestion instead of creating a new component to view the studio assessment?